datafusion/physical-expr/src/string_expressions.rs

```diff
@@ -476,6 +520,24 @@ pub fn starts_with<T: OffsetSizeTrait>(args: &[ArrayRef]) -> Result<ArrayRef> {
     Ok(Arc::new(result) as ArrayRef)
 }
 
+/// Returns true if string ends with suffix.
+/// ends_with('alphabet', 'abet') = 't'
+pub fn ends_with<T: OffsetSizeTrait>(args: &[ArrayRef]) -> Result<ArrayRef> {
+    let string_array = as_generic_string_array::<T>(&args[0])?;
```

Comment:
We should be able to use https://docs.rs/arrow/latest/arrow/compute/kernels/comparison/fn.ends_with.html

Reply:
Do you mean that I can call this library function directly in the code or do I not need to implement this function?

Reply:
> Do you mean that I can call this library function directly in the code or do I not need to implement this function?

Yes, I think  the body of thus function could be reduced to something like

```suggestion
    compute::comparison::ends_with(&args[0])
```